extension/k8s/demo/values.yaml

```diff
@@ -0,0 +1,11 @@
+nonSensitiveEnvs:
+  CTP_PROJECT_KEY: adyen-integration-demo
```

Comment:
You have this var twice - in secrets and here. Check also notification module

Reply:
fe699ba853a0f361885cf7460e6ea2bc8ba7321b